examples/with-nextjs-and-clerk-auth/src/monite-theme.d.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import '@mui/material/styles';
+import '@mui/x-data-grid/themeAugmentation';
+
+declare module '@mui/material/styles' {
+  interface TypeBackground {
+    default: string;
+    paper: string;
+    menu: string;
+    highlight: string;
+  }
```

Comment:
It doesn't make sense to me to add all these types just for the sole purpose of using it in a single place:
```ts
PaperProps={{ sx: { backgroundColor: 'background.menu' } }}
```

Furthermore, if you want to change the styles for all menus, why not do it? Why style menus in Demo one way and in SDK intentionally another way? It seems like the idea of Demo is to show that widgets can be styled any way you want.
I recommend focusing on customizing the theme MUI than creating new entities that require complex support.

Reply:
Just the opposite. I would like SDK to be able to reuse all styles from Demo as much as possible. Ideally - it would not be clear where Demo ends and SDK begins.
As for `<Drawer/>` - you can style it globally, through the theme. Especially since it is not used in the SDK (and if it was - it would be cool to get similar colors for it).